WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfilesViewModel.cs

```diff
@@ -0,0 +1,76 @@
+using ReactiveUI;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Fluent.ViewModels.AddWallet;
+using WalletWasabi.Fluent.ViewModels.Dialogs;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinJoinProfiles;
+
+[NavigationMetaData(Title = "Coinjoin Strategy")]
+public partial class CoinJoinProfilesViewModel : RoutableViewModel
+{
+	[AutoNotify] private CoinJoinProfileViewModelBase? _selectedProfile;
+
+	public CoinJoinProfilesViewModel(KeyManager keyManager)
+	{
+		NextCommand = ReactiveCommand.Create(() => OnNext(keyManager));
+		EnableBack = true;
+
+		var speedyProfile = new SpeedyCoinJoinProfile();
+
+		Profiles = new()
+		{
+			new EconomyCoinJoinProfile(),
+			speedyProfile,
+			new PrivateCoinJoinProfile()
+		};
+
+		_selectedProfile = speedyProfile;
+
+		ManualSetupCommand = ReactiveCommand.CreateFromTask(async () => await OnManualSetupAsync());
+	}
+
+	public ICommand ManualSetupCommand { get; }
+
+	public List<CoinJoinProfileViewModelBase> Profiles { get; }
+
+	public ManualCoinJoinProfile? SelectedManualProfile { get; private set; }
+
+	private async Task OnManualSetupAsync()
+	{
+		var current = SelectedProfile ?? SelectedManualProfile ?? Profiles.First();
+		var dialog = new ManualCoinJoinProfileDialogViewModel(current);
+
+		var dialogResult = await NavigateDialogAsync(dialog, NavigationTarget.CompactDialogScreen);
+
+		if (dialogResult.Result is ManualCoinJoinProfile profile)
+		{
+			SelectedProfile = null;
+			SelectedManualProfile = profile;
```

Comment:
You can just simply set `SelectedProfile = profile`, thus you can remove `SelectedManualProfile` completely and `SelectedProfile` doesn't need to be nullable so you can remove all the null checks.

Reply:
It is not working. It seems like it the item is not in the list, then right after setting SelectedProfile it became null.

Reply:
I don't understand it. It should work https://github.com/soosr/WalletWasabi/commit/66b4ca6b839123b88623e915cff41238d61dbf0a

Reply:
It was because of the binding. Using one-way binding solves it. https://github.com/soosr/WalletWasabi/tree/simplify-selection